frontend/src/components/LogVisualizer.jsx

```diff
@@ -73,112 +79,123 @@ const buildLineElems = (
+    const exportName = getUrlSafeProjectNameFull(project);
+    // eslint-disable-next-line react/no-find-dom-node
+    downloadChartAsPng(ReactDOM.findDOMNode(this.chart), exportName);
+  }
+
+  render() {
+    const {
+      project = {},
+      results = {},
+      projectConfig = {},
+      globalConfig = {},
+      stats
+    } = this.props;
+    const { axes, resultsConfig = {} } = projectConfig;
+    const { logKeys = [], xAxisKeys } = stats;
+    const {
+      xAxis = { axisName: 'xAxis' },
+      yLeftAxis = { axisName: 'yLeftAxis' },
+      yRightAxis = { axisName: 'yRightAxis' }
+    } = axes || {};
+    const { xAxisKey = xAxisKeys[0] } = xAxis;
+    const selectedResults = getSelectedResults(results, resultsConfig);
+    const selectedLogKeys = {
+      yLeftAxis: getSelectedLogKeys(yLeftAxis.logKeysConfig),
+      yRightAxis: getSelectedLogKeys(yRightAxis.logKeysConfig)
+    };
+
+    const data = getLogData(results, stats, projectConfig);
+
+    const lineElems = [
+      ...buildLineElems(selectedResults, selectedLogKeys.yLeftAxis, 'yLeftAxis', results, projectConfig, logKeys),
+      ...buildLineElems(selectedResults, selectedLogKeys.yRightAxis, 'yRightAxis', results, projectConfig, logKeys)
+    ];
+
+    const { chartSize, isResultNameAlignRight } = globalConfig;
+
+    return (
+      <div className="log-visualizer-root">
+        <ResponsiveContainer
+          width={chartSize.width}
+          height={chartSize.height}
+          aspect={chartSize.aspect}
+        >
+          <LineChart data={data} ref={this.chartRef}>
+            <XAxis
+              type="number"
+              dataKey={xAxisKey}
+              scale={xAxis.scale}
+              domain={getDomain(xAxis)}
+              allowDataOverflow
+            />
+            <YAxis
+              yAxisId="yLeftAxis"
+              orientation="left"
+              scale={yLeftAxis.scale}
+              domain={getDomain(yLeftAxis)}
+              tickFormatter={formatLogValue()}
+              allowDataOverflow
+            />
+            <YAxis
+              yAxisId="yRightAxis"
+              orientation="right"
+              scale={yRightAxis.scale}
+              domain={getDomain(yRightAxis)}
+              tickFormatter={formatLogValue()}
+              allowDataOverflow
+            />
+            <CartesianGrid strokeDasharray="3 3" />
+            <Tooltip
+              content={
+                <LogVisualizerTooltip
+                  project={project}
+                  results={results}
+                  xAxisKey={xAxisKey}
+                  isResultNameAlignRight={isResultNameAlignRight}
+                />
             }
-          />
-          {lineElems}
-        </LineChart>
-      </ResponsiveContainer>
-    </div>
-  );
-};
+            />
+            {lineElems}
+          </LineChart>
+        </ResponsiveContainer>
+        <div>
+          <Button size="xs" className="m-1" onClick={this.handleClickDownloadJSON}>
```

Comment:
`size="xs"` isn't available in Bootstrap 4.

cf. https://getbootstrap.com/docs/4.1/components/buttons/#sizes